lib/srv/app/server.go

```diff
@@ -440,6 +441,7 @@ func (s *Server) newHTTPServer() *http.Server {
 	return &http.Server{
 		Handler:           authMiddleware,
 		ReadHeaderTimeout: defaults.DefaultDialTimeout,
+		ErrorLog:          stdlog.New(utils.NewStdlogAdaptor(s.log.Error), teleport.ComponentApp, stdlog.LstdFlags),
```

Comment:
this looks like a common pattern.
how about pulling it into the utils package, something like
```
func NewStdlog(logger leveledOutputFunc, component string) *stdlog.Logger
```